Gems/MotionMatching/Code/Tests/MinMaxScalerTests.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Fixture.h>
+#include <FeatureMatrixMinMaxScaler.h>
+
+namespace EMotionFX::MotionMatching
+{
+    class MinMaxScalerFixture
+        : public Fixture
+    {
+    public:
+        void SetUp() override
+        {
+            Fixture::SetUp();
+
+            // Construct 3x3 matrix:
+            // 1 2 3
+            // 4 5 6
+            // 7 8 9
+            m_featureMatrix.resize(3, 3);
+
+            float counter = 1.0f;
+            for (size_t row = 0; row < 3; ++row)
+            {
+                for (size_t column = 0; column < 3; ++column)
+                {
+                    m_featureMatrix(row, column) = counter;
+                    counter++;
+                }
+            }
+        }
+
+        FeatureMatrix m_featureMatrix;
+    };
+
+    TEST_F(MinMaxScalerFixture, MinMaxValues)
+    {
+        FeatureMatrix m;
+        m.resize(3, 3);
+        m(0,0) = 0.0f;    m(0,1) =-1.0f;    m(0,2) = 9.0f;
+        m(1,0) = 0.5f;    m(1,1) = 5.0f;    m(1,2) = 6.0f;
+        m(2,0) = 7.0f;    m(2,1) = 0.1f;    m(2,2) = 3.0f;
+
+        MinMaxScaler minMaxScaler;
+        EXPECT_TRUE(minMaxScaler.Fit(m));
+
+        const AZStd::vector<float>& min = minMaxScaler.GetMin();
+        const AZStd::vector<float>& max = minMaxScaler.GetMax();
+        EXPECT_EQ(min[0], 0.0f); EXPECT_EQ(max[0], 7.0f);
```

Comment:
I would suggest using EXPECT_NEAR rather than EXPECT_EQ, otherwise you might get things like 2.000001f != 2.0f.